diktat-test-framework/src/main/kotlin/org/cqfn/diktat/test/framework/processing/FileComparator.kt

```diff
@@ -47,7 +47,7 @@ class FileComparator {
 |                        See difference below:
 |                           Expected  vs  Actual ${System.lineSeparator()}$deltasJoiner""".trimMargin())
 
-        } catch (e: DiffException) {
+        } catch (e: RuntimeException) {
```

Comment:
RuntimeException? Are you sure? Looks not ok

Reply:
I found comment by library author related to this change: https://github.com/java-diff-utils/java-diff-utils/issues/70#issuecomment-595988726
where they say that parsing library shouldn't throw any checked exceptions. Do you think we should catch `Exception` here or just remove `try-catch`?

Reply:
Never try to catch Exception. I just thought that they have added some custom RuntimeException and you will catch it here. If not - lgtm